package.json

```diff
@@ -1,947 +1,947 @@
 {
```

Comment:
Please revert changes to this file - looks like only whitespace changes. FWIW - I've filed https://github.com/microsoft/vscode-cosmosdb/issues/1235 to clean up whitespace in this repo and add VS Code settings to automatically keep it consistent